src/v/kafka/protocol/schemata/generator.py

```diff
@@ -697,6 +697,9 @@ def tag(self):
     def nullable(self):
```

Comment:
- Previously our generator observed any type as 'nullable' if the field
existed.

I'm not really sure what this means. Would an example here help like in the previosu commit?